dev/preview/workflow/preview/post-process.sh

```diff
@@ -179,94 +158,7 @@ while [ "$documentIndex" -le "$DOCS" ]; do
-
-      WS_PORT_URL_TEMP_EXPR="s|\"portUrlTemplate\": \"https://{{ .WorkspacePort }}-{{ .Prefix }}.$CURRENT_WS_HOST_NAME\"|\"portUrlTemplate\": \"https://{{ .WorkspacePort }}-{{ .Prefix }}.$NEW_WS_HOST_NAME\"|"
-      sed -i "$WS_PORT_URL_TEMP_EXPR" /tmp/"$NAME"overrides.yaml
-
-      WS_URL_TEMP_EXPR="s|\"urlTemplate\": \"https://{{ .Prefix }}.$CURRENT_WS_HOST_NAME\"|\"urlTemplate\": \"https://{{ .Prefix }}.$NEW_WS_HOST_NAME\"|"
-      sed -i "$WS_URL_TEMP_EXPR" /tmp/"$NAME"overrides.yaml
-
-      # Change the port we use to connect to registry-facade
-      # is expected to be reg.<branch-name-with-dashes>.staging.gitpod-dev.com:$REG_DAEMON_PORT
-      # Change the port we use to connect to ws-daemon
-      REGISTRY_FACADE_HOST="reg.$DEV_BRANCH.staging.gitpod-dev.com:$REG_DAEMON_PORT"
-      if [[ -v WITH_VM ]]; then
-         REGISTRY_FACADE_HOST="reg.$DEV_BRANCH.preview.gitpod-dev.com:$REG_DAEMON_PORT"
-      fi
-      yq r /tmp/"$NAME"overrides.yaml 'data.[config.json]' \
-      | jq --arg REGISTRY_FACADE_HOST "$REGISTRY_FACADE_HOST" '.manager.registryFacadeHost = $REGISTRY_FACADE_HOST' > /tmp/"$NAME"-cm-overrides.json
-
-      touch /tmp/"$NAME"-cm-overrides.yaml
-      # write a yaml file with the json as a multiline string
-      yq w -i /tmp/"$NAME"-cm-overrides.yaml "data.[config.json]" -- "$(< /tmp/"$NAME"-cm-overrides.json)"
-      yq m -x -i /tmp/"$NAME"overrides.yaml /tmp/"$NAME"-cm-overrides.yaml
-
-      yq m -x -i k8s.yaml -d "$documentIndex" /tmp/"$NAME"overrides.yaml
-   fi
-
-    # overrides for ws-manager-mk2
-   if [[ "ws-manager-mk2" == "$NAME" ]] && [[ "$KIND" == "ConfigMap" ]]; then
-      WORK="overrides for $NAME $KIND"
-      echo "$WORK"
-       # Change the port we use to connect to registry-facade
-      # is expected to be reg.<branch-name-with-dashes>.staging.gitpod-dev.com:$REG_DAEMON_PORT
-      # Change the port we use to connect to ws-daemon
-      REGISTRY_FACADE_HOST="reg.$DEV_BRANCH.staging.gitpod-dev.com:$REG_DAEMON_PORT"
-      if [[ -v WITH_VM ]]; then
-         REGISTRY_FACADE_HOST="reg.$DEV_BRANCH.preview.gitpod-dev.com:$REG_DAEMON_PORT"
-      fi
-
-      # get a copy of the config we're working with
-      yq r k8s.yaml -d "$documentIndex" > /tmp/"$NAME"-"$KIND"-overrides.yaml
-
-      # replace registry port
-      yq r /tmp/"$NAME"-"$KIND"-overrides.yaml 'data.[config.json]' \
-      | jq ".manager.registryFacadeHost = \"$REGISTRY_FACADE_HOST\"" > /tmp/"$NAME"-"$KIND"-overrides.json
-
-      # create override file
-      touch /tmp/"$NAME"-"$KIND"-data-overrides.yaml
-      yq w -i /tmp/"$NAME"-"$KIND"-data-overrides.yaml "data.[config.json]" -- "$(< /tmp/"$NAME"-"$KIND"-overrides.json)"
-
-      # merge the updated config map with k8s.yaml
-      yq m -x -i k8s.yaml -d "$documentIndex" /tmp/"$NAME"-"$KIND"-data-overrides.yaml
-   fi
-
-   # overrides for ws-proxy
-   if [[ "ws-proxy" == "$NAME" ]] && [[ "$KIND" == "ConfigMap" ]]; then
```

Comment:
Why remove ws-proxy changes? Just curious if it was breaking something, or no longer needed/relevant.

Reply:
I see, so, this is just clean-up